server/log.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+/**
+ * redis.js
+ *
+ * Used to store Audit logs.
+ */
+
+const Redis = require("redis");
+const logLength = 5000;
```

Comment:
We already do logs to LogEntries and have done for a long time. But if people used them then clearly this ticket would not be needed. Hence Redis as requested.
It's a log of who recently made mischief if old records are required log entries can be used.